BezierSwift/Sources/Components/Checkbox/BezierPrimaryCheckbox.swift

```diff
@@ -0,0 +1,217 @@
+    label: String?,
+    color: Color,
+    checked: Checked,
+    showRequired: Bool,
+    @ViewBuilder nestedBuilder: @escaping NestedBuilder
+  ) {
+    if let label {
+      self.label = AttributedString(label)
+    } else {
+      self.label = nil
+    }
+    self.color = color
+    self.checked = checked
+    self.showRequired = showRequired
+    self.nestedBuilder = nestedBuilder
+  }
+
+  /// Nested가 없는 체크박스를 생성합니다.
+  /// - Parameters:
+  ///   - label: 체크박스에 함께 표기될 텍스트를 지정합니다.
+  ///   - color: 체크박스 소스가 표기될 색상을 지정합니다. blue와 green을 지정할 수 있습니다.
+  ///   - checked: 체크박스의 체크 상태를 지정합니다. checked, unchecked, indeterminate를 지정할 수 있습니다.
+  ///   - showRequired: * 표시(Asterisk)를 사용해서 필수 항목임을 표현할지 지정합니다.
+  public init(
+    label: String?,
+    color: Color,
+    checked: Checked,
+    showRequired: Bool
+  ) where Nested == EmptyView {
+    if let label {
+      self.label = AttributedString(label)
+    } else {
+      self.label = nil
+    }
+    self.color = color
+    self.checked = checked
+    self.showRequired = showRequired
+    self.nestedBuilder = { EmptyView() }
+  }
+  
+  /// Nested가 존재하는 체크박스를 생성합니다.
+  /// - Parameters:
+  ///   - label: 체크박스에 함께 표기될 AttributedString을 지정합니다.
+  ///   - color: 체크박스 소스가 표기될 색상을 지정합니다. blue와 green을 지정할 수 있습니다.
+  ///   - checked: 체크박스의 체크 상태를 지정합니다. checked, unchecked, indeterminate를 지정할 수 있습니다.
+  ///   - showRequired: * 표시(Asterisk)를 사용해서 필수 항목임을 표현할지 지정합니다.
+  ///   - nestedBuilder: 체크박스의 하위 영역에 표시될 내용을 지정하는 뷰를 생성합니다.
+  public init(
+    label: AttributedString?,
+    color: Color,
+    checked: Checked,
+    showRequired: Bool,
+    @ViewBuilder nestedBuilder: @escaping NestedBuilder
+  ) {
+    self.label = label
+    self.color = color
+    self.checked = checked
+    self.showRequired = showRequired
+    self.nestedBuilder = nestedBuilder
+  }
+
+  /// Nested가 없는 체크박스를 생성합니다.
+  /// - Parameters:
+  ///   - label: 체크박스에 함께 표기될 AttributedString을 지정합니다.
+  ///   - color: 체크박스 소스가 표기될 색상을 지정합니다. blue와 green을 지정할 수 있습니다.
+  ///   - checked: 체크박스의 체크 상태를 지정합니다. checked, unchecked, indeterminate를 지정할 수 있습니다.
+  ///   - showRequired: * 표시(Asterisk)를 사용해서 필수 항목임을 표현할지 지정합니다.
+  public init(
+    label: AttributedString?,
+    color: Color,
+    checked: Checked,
+    showRequired: Bool
+  ) where Nested == EmptyView {
+    self.label = label
+    self.color = color
+    self.checked = checked
+    self.showRequired = showRequired
+    self.nestedBuilder = { EmptyView() }
+  }
+
+  public var body: some View {
+    VStack(spacing: 0) {
+      HStack(alignment: .top, spacing: 0) {
+        ZStack(alignment: .center) {
+          BezierCheckboxSource(
+            needStroke: self.sourceNeedStroke,
+            icon: self.sourceIcon,
+            strokeColor: self.sourceStrokeColor,
+            backgroundColor: self.sourceBackgroundColor,
+            iconColor: self.sourceIconColor
+          )
+        }
+        .frame(length: Metric.minHeight)
+
+        HStack(alignment: .center, spacing: 0) {
```

Comment:
```suggestion
        HStack(alignment: .center, spacing: .zero) {
```
채널 엑스에서는 spacing 0이 들어가는 경우가 많아서 이때마다 Metric에 추가하기 어려워서 0 일 때만 .zero로 구분해주자 했었어요. 이 컨벤션을 베지어에서도 가져갈지 논의 해보면 좋을 것 같아요